scripts/bump-version.sh

```diff
@@ -10,12 +10,12 @@ NEW_VERSION="${2}"
 echo "Current version: ${OLD_VERSION}"
 echo "Bumping version: ${NEW_VERSION}"
 
-VERSION_RE=${OLD_VERSION//\./\\.}
-find . -name Cargo.toml -not -path './relay-cabi/*' -exec sed -i.bak -e "1,/^version/ s/^version.*/version = \"${NEW_VERSION}\"/" {} \;
-find . -name Cargo.toml.bak -exec rm {} \;
+TOML_FILES="$(git ls-files '*Cargo.toml' | grep -v cabi)"
```

Comment:
This uses `git ls-files` now as a faster alternative to `find` by ignoring the ever-growing `target` directory.

Reply:
Is this needed for macOS compat or just an optimization while you were passing by?

Reply:
This is just an optimization, and the perl below is to streamline with other repos, like [`sentry-rust`](https://github.com/getsentry/sentry-rust/blob/36218ea5e815d871c073ee15d7f71ee01dab69c7/scripts/bump-version.sh#L13).